web-client/src/models/requests/RequestWithOffersAndTimeline.ts

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable @typescript-eslint/no-explicit-any */
```

Comment:
Renamed timeline to timeline items.  Might have been a waste of time, I just kept getting annoyed timeline was not plural.